scss/mixins/_list-group.scss

```diff
@@ -1,3 +1,6 @@
+@include deprecate("`list-group-item-variant()`", "v5.3.0", "v6.0.0");
+
+
```

Comment:
Need more space in here ? :smile:

Reply:
Already fixed in Bootstrap in another PR (see https://github.com/twbs/bootstrap/pull/37829/files). Voluntarily let it in this PR.